qiita_pet/portal.py

```diff
@@ -0,0 +1,94 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from os import environ
+from os.path import dirname, abspath, join
+from future import standard_library
+
+from qiita_core.exceptions import MissingConfigSection
+
+with standard_library.hooks():
+    from configparser import ConfigParser
+
+
+class PortalStyleManager(object):
+    """Holds the portal style information
+
+    Parameters
+    ----------
+    conf_fp: str, optional
+        Filepath to the configuration file. Default: portal.txt
+
+    Attributes
+    ----------
+    logo : str
+        Path from base URL to the site logo
+    title : str
+        Site title
+    navbar_color : str
+        Hex or text color for navbar background color
+    navbar_highlight : str
+        Hex or text color for navbar highlight of selected menu
+    navbar_text_color : str
+        Hex or text color for navbar menu text
+    navbar_text_hover : str
+        Hex or text color for navbar hover over for menu text
+    text_bg : str
+        Hex or text color for index jumbotron background
+    text_color : str
+        Hex or text color for index welcome text
+    index header : str
+        Welcome text header for the website
+    index_text : str
+        Welcome text for the website
+    example_search : str
+        Example search to be shown on the study listing page
+    """
+    def __init__(self, conf_fp=None):
+        # If conf_fp is None, we default to the test configuration file
+        try:
+            conf_fp = environ['QIITA_PORTAL_FP']
```

Comment:
I don't agree with this one. Just add another key to the current configuration, like you added portal.